packages/core/src/utils/CredentialResponseCoordinator.ts

```diff
@@ -1,24 +1,13 @@
 import { scoped, Lifecycle } from 'tsyringe'
 
-import { AgentConfig } from '../../agent/AgentConfig'
-import { DidCommMessageRepository } from '../../storage'
-
-import { AutoAcceptCredential } from './CredentialAutoAcceptType'
+import { AutoAcceptCredential } from '../modules/credentials/CredentialAutoAcceptType'
 
 /**
  * This class handles all the automation with all the messages in the issue credential protocol
  * Every function returns `true` if it should automate the flow and `false` if not
  */
 @scoped(Lifecycle.ContainerScoped)
 export class CredentialResponseCoordinator {
```

Comment:
What I meant with making it an util is to remove the whole class. The class used to have a lot of methods, but now it's just a simple function, so I don't see the need for it anymore. We can just call `composeAutoAccept` as a function instead of as `CredentialResponseCoordinator.composeAutoAccept`. It doesn't coordinate anything anymore